src/include/block_chunkcache.h

```diff
@@ -50,6 +50,8 @@ struct __wt_chunkcache_chunk {
     TAILQ_ENTRY(__wt_chunkcache_chunk) next_lru_item;
 
     WT_CHUNKCACHE_HASHID hash_id;
+
+#define WT_CHUNK_ACCESS_CAP_LIMIT 1000
```

Comment:
This could be an undocumented config option if we wish to experiment with this in the future. Thoughts?

Reply:
As this access count is per chunk, it does seem pretty high. I agree with doing testing to see what is a reasonable number.

Reply:
It has been challenging to test this. Initially, I wanted to choose a lower number (like 10). However, the intention behind opting for a higher number is that it allows eviction to take its time in clearing obsolete content, allowing an opportunity (and enough time) for parallel reader operations to increment the `access_count` of the chunks in use. 
There is a possibility of over-eviction when (majority) chunks reach the maximum limit (10), which seems likely to occur, and such a scenario could result in mass eviction, given the current logic. However, we can address this issue by implementing a mechanism that limits the amount of eviction per round. For instance, we could ensure that no more than 15% of the cache becomes empty during each round of eviction.

Reply:
<img width="1476" alt="Screenshot 2023-12-04 at 1 41 50 pm" src="https://github.com/wiredtiger/wiredtiger/assets/55784503/9e1014da-3746-424d-aed2-117e97fe5e6c">
Ran format and the eviction rate looks quite consistent, with no mass eviction. I have run a patch with 100 iterations.

Reply:
> There is a possibility of over-eviction when (majority) chunks reach the maximum limit

Fair point - let's leave it at 1000 and see how it plays out.